### PR TITLE
kd/auth: do not overwrite baseurl / ensure klient is fetched from managed channel

### PR DESCRIPTION
**Pull request by rjeczalik**

### CODE REVIEW

File: go/src/koding/klientctl/endpoint/auth/auth.go [hunk 1]
```diff
@@ -44,10 +44,11 @@ type LoginOptions struct {
 	Token    string
 	Username string
 	Password string
+	Prefix   string
 }
 
 func (opts *LoginOptions) AskUserPass() (err error) {
-	opts.Username, err = helper.Ask("Username [%s]: ", config.CurrentUser.Username)
+	opts.Username, err = helper.Ask("%sUsername [%s]: ", opts.Prefix, config.CurrentUser.Username)
```
Comment (ppknap) [inline go/src/koding/klientctl/endpoint/auth/auth.go]: Just out of curiosity, how `opts.Prefix` can look like? Will it be `prefixUsername [user]` or `PrefixUsername [user]` ?

**Review by ppknap — APPROVED** [review]

File: go/src/koding/klientctl/endpoint/auth/auth.go (hunk 1 above)
Comment (rjeczalik) [inline go/src/koding/klientctl/endpoint/auth/auth.go]: At the moment prefix is used in daemon/install to apply proper indentation to the output.

**Comment by rjeczalik** [comment]
Please do not merge yet, I'm fixing build failures.

**Comment by rjeczalik** [comment]
Done, ptal.

**Review by ppknap — APPROVED** [review]

**Comment by rjeczalik** [comment]
@cihangir Ptal, I'm going to address your comments if you have any in a separate PR.

**Comment by cihangir** [comment]
👍

**Pull request merged**